### ISSUE TITLE
RC Test: Fraction Matcher 1.1.0-rc.1

### ISSUE THREAD

**Issue by samreid** [phetsims/tasks#541]
Fraction Matcher 1.1.0-rc.1 is ready for testing (as created in https://github.com/phetsims/fraction-matcher/issues/92 )

**Link to sim:** http://www.colorado.edu/physics/phet/dev/html/fraction-matcher/1.1.0-rc.1/fraction-matcher_en.html				

**Link to iFrame:** http://www.colorado.edu/physics/phet/dev/html/fraction-matcher/1.1.0-rc.1/fraction-matcher_en-iframe.html

**Test Matrix:** https://docs.google.com/spreadsheets/d/1AZhRl162qczy4-1BuTQPfB1mMwQ7t0AcRR-L1PWepKg/edit#gid=2138290843

Please do a full retest of this sim to make sure that no new issues were introduced.  This is also the first release candidate with support for internationalization, so please use ?stringTest appropriately. If you notice any issues that are not listed in https://github.com/phetsims/fraction-matcher/issues, please create a new issue and reference this one.

**Redeploy General Tests** (copied from https://github.com/phetsims/tasks/issues/528 )
The following tests should also be performed as part of this testing
- [x] stringTest=double
- [x] stringTest=long
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (should not redirect, OK to look bad, test on one desktop platform)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Reference issues to RC test task

@ariel-phet please make sure I've provided all the necessary details (it has been a long time since my last RC test), prioritize and reassign.

**Comment by ariel-phet** [comment]
@BryceAG @brroberts1231 @elisemorgan @bryo5363 @phet-steele please makes this the current next testing priority after #540 is complete thanks!

**Comment by ariel-phet** [comment]
assigning to @phet-steele to move forward

**Comment by elisemorgan** [comment]
Testing complete

**Comment by samreid** [comment]
Thanks for testing, everyone, I'll continue in https://github.com/phetsims/fraction-matcher/issues/92
